src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
-body {
+body,
+html {
   margin: 0;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$calendar-size: 100px;
+$calendar-gap: $calendar-size / 0.01;
```

Comment:
The calculation of `$calendar-gap` seems incorrect. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the gap should be a reasonable value, but dividing by 0.01 is the same as multiplying by 100, which would give an extremely large gap. Consider setting a fixed value or a more reasonable formula.